mlir/include/mlir/Dialect/GPU/GPUOps.td

```diff
@@ -973,6 +973,7 @@ def GPU_MubufStoreOp :
                               VectorOfLengthAndType<[2, 4], [F32]>,
                               VectorOfLengthAndType<[2, 4, 8], [F16, I16]>]>:$value,
                    AnyMemRef:$memref,
+                   I32:$shift,
```

Comment:
please change the `description` of this op to briefly explain what this `shift` is. what should be the normal value, what should be the value when we want to do OOB store.